### PR TITLE
Fix wrong menu link

### LINKED ISSUE #814: Wrong link to scraper overview in navigation
The link to technical overview points to scraper-overview.html which does not exist.

### PR DESCRIPTION
**Pull request by s-pace**
resolves #814

### CODE REVIEW

File: docs/config.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
       "pages": [
         {
           "title": "DocSearch Technical Overview (the Scraper)",
-          "url": "scraper-overview.html"
+          "url": "crawler-overview.html"
```
Comment (Haroenv) [inline docs/config.json]: don't you want to change the page's name instead?

Comment (s-pace) [reply docs/config.json]: I think it should be enough

Comment (Haroenv) [reply docs/config.json]: i mean changing the file instead of the url in the config :)

Comment (s-pace) [reply docs/config.json]: Let’s keep it this way

**Pull request merged**